frontend/src/metabase/visualizations/lib/settings/graph.unit.spec.js

```diff
@@ -286,4 +281,79 @@ describe("GRAPH_DISPLAY_VALUES_SETTINGS", () => {
       expect(isHidden).toBe(false);
     });
   });
+
+  describe("graph.show_stack_values", () => {
+    const getHidden =
+      GRAPH_DISPLAY_VALUES_SETTINGS["graph.show_stack_values"].getHidden;
+
+    it("should be hidden on non-stacked charts", () => {
+      const isHidden = getHidden(
+        [{ card: { display: "bar" } }, { card: { display: "bar" } }],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stack_type": null,
+          "graph.show_values": true,
+        },
+      );
+
+      expect(isHidden).toBe(true);
+    });
+
+    it("should be hidden on stacked area charts", () => {
+      const isHidden = getHidden(
+        [{ card: { display: "area" } }, { card: { display: "area" } }],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stack_type": "stacked",
+          "graph.show_values": true,
+        },
+      );
+
+      expect(isHidden).toBe(true);
+    });
+
+    it("should not be hidden on mixed stacked area and bar charts", () => {
+      const isHidden = getHidden(
+        [
+          { card: { display: "area" } },
+          { card: { display: "area" } },
+          { card: { display: "bar" } },
+          { card: { display: "bar" } },
+        ],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stack_type": "stacked",
+          "graph.show_values": true,
+        },
+      );
+
+      expect(isHidden).toBe(false);
+    });
+
+    it("should be hidden on normalized charts bar charts", () => {
+      const isHidden = getHidden(
+        [{ card: { display: "bar" } }, { card: { display: "bar" } }],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stack_type": "normalized",
+          "graph.show_values": true,
+        },
+      );
+
+      expect(isHidden).toBe(true);
+    });
```

Comment:
If this is hidden on normalized bar charts (which I believe makes sense), we need to act as if the value is set to `"total"` because otherwise the `Show values on data points` setting does not make sense. It does not allow the user to configure whether they want inside labels or not in `Stack - 100%`.

But also considering the case where we have a normalized bar chart with a regular line series added, we should still allow the inside labels to be configurable. Like still show the setting but only show valid options for it like `None` or `Insides`.

![image](https://github.com/metabase/metabase/assets/22608765/804bdf6a-9d04-486e-8eee-ea4e98e16fe0)

Reply:
Also I know the product doc displays the labels with the absolute values, but do we want to instead to display the relative percentages in a normalized bar chart (`Stack - 100%`). To me this makes more sense to show, because the user intentionally selected normalized, but we are still showing absolute values?

![image](https://github.com/metabase/metabase/assets/22608765/0f13aade-71b2-4845-bff6-4e2d41b5407e)